cachi2/core/package_managers/yarn_classic/workspaces.py

```diff
@@ -0,0 +1,23 @@
+import json
```

Comment:
**Commit subject:**

s/yar/yarn

Reply:
**Commit message:**

To make it more useful please mention what constitutes a workspace which lead us to this representation.

Reply:
Done